modules/compliance-operator-cli-installation.adoc

```diff
@@ -9,6 +9,7 @@
 .Prerequisites
 
 * You must have `admin` privileges.
+* You must have `StorageClass` configured.
```

Comment:
There are 5 instances of "`StorageClass`" across the PR that could probably be "a `StorageClass` resource" or similar.